api/Headers.json

```diff
@@ -0,0 +1,1106 @@
+{
+  "api": {
+    "Headers": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Headers",
+        "support": {
+          "webview_android": {
+            "version_added": "42"
+          },
+          "chrome": {
+            "version_added": "42"
+          },
+          "chrome_android": {
+            "version_added": "42"
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": [
+            {
+              "version_added": "52",
```

Comment:
Remove the entry with 52 and the note. This belongs to the `get()` entry and not here.

Reply:
👍